ci-infra/azure/outputs.tf

```diff
@@ -33,3 +33,7 @@ output "AZURE_PODVM_IMAGE_DEF_NAME" {
 output "AZURE_MANAGED_IDENTITY_NAME" {
   value = azurerm_user_assigned_identity.gh_action_user_identity.name
 }
+
+output "AZURE_COMMUNITY_GALLERY_NAME" {
+  value = azurerm_shared_image_gallery.podvm_image_gallery.sharing[0].community_gallery[0].name
+}
```

Comment:
We need to get these terraform changes in, before we merge the workflow. Because without the terraform changes, the workflow will publish an image but it won't be public. Which should not fail per se, but then we have to run the terraform commands before UTC midnight 😆

Reply:
Sooo terraform commit will get merged first if we merge this whole PR that should suffice right ?

Reply:
I think it is ok, we can merge it together. We have a small window before the workflow runs.